frontend/src/pages/org/collection-detail.ts

```diff
@@ -409,37 +499,18 @@ export class CollectionDetail extends BtrixElement {
                 await this.updateComplete;
               }
 
-              this.openDialogName = "editStartPage";
+              this.openDialogName = "replaySettings";
```

Comment:
Is this intended to be for editing the initial view or a repeat of the general edit collection button? If the former, the label will need to be updated. If latter, open dialog name should be `edit`.